about.css

```diff
@@ -0,0 +1,215 @@
+.about-headline {
+  background: url('./resources/image/climbers.jpeg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  min-height: 92vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-color: #d7cfcf;
+  padding: 20px;
+  background-blend-mode: color-dodge;
+  overflow: hidden;
+}
+
+.headline-block {
+  max-width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 70px;
+}
+
+.hero-greet {
+  font-size: 28px;
+  text-align: center;
+}
+
+.headline-title {
+  font-size: 32px;
+  font-weight: 900;
+  color: #eb1d07;
+  margin: 30px 0;
+  text-align: center;
+  line-height: 35px;
+}
+
+.headline-details {
+  background: #fff;
+  color: #292d35;
+  padding: 25px;
+  max-width: 100%;
+  border: 1px solid #cdcbcb;
+  margin-bottom: 15px;
+  text-align: center;
+}
+
+.headline-block h5 {
+  font-weight: normal;
+  color: #292c34;
+  margin: 20px 0;
+  text-align: center;
+}
+
+address {
+  text-decoration: underline;
+  font-style: normal;
+  font-weight: bold;
+  margin-bottom: 20px;
+}
+
+.about-logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 25px;
+  background: #fff;
+  flex-direction: column;
+  border-bottom: 1px solid #d6d6d6;
+}
```

Comment:
- [ Required ] I think it would be better if you make sure that the `margins` and `paddings` in these parts of the `About` page match those in the Behance design as requested by a previous reviewer. Currently, it appears that your design has much wider `margins` and `paddings` compared to the Behance design. Please see the screenshot below and rectify this.

## Behance design
![Screenshot from 2023-07-07 04-09-22](https://github.com/Uzair-Manzoor/Capstone/assets/16392046/00489cf7-2a00-4f2c-ac1e-8004e01a1e15)

## Your design
![Screenshot from 2023-07-07 04-06-40](https://github.com/Uzair-Manzoor/Capstone/assets/16392046/4aa6625a-6b9d-49c2-8697-0ce5db426866)